crawling/main/main.py

```diff
@@ -0,0 +1,7 @@
+from crawling.main.crawling import Crawling
+
+# 여기서 컴퓨팅 리소스를 더 쓰되, 두 크롤링을 동시에 수행하게 만든다면 더욱 재밌
+if __name__ == '__main__':
+    # DO create table
+    Crawling.main_page_crawling("https://www.musinsa.com/app/styles/lists")
+    Crawling.main_page_crawling("https://www.musinsa.com/app/codimap/lists")
```

Comment:
정의한 `Crawling` class에 맞게 객체를 아래 예시와 같이 선언 하여 사용해야겠네요.
`main_page_crawling` 함수는 argument가 따로 없어 보이네요.
```suggestion
    crawling_styles = Crawling(top_url="https://www.musinsa.com/app/styles/lists")
    crawling_codimap = Crawling(top_url="https://www.musinsa.com/app/codimap/lists")
    crawling_styles.main_page_crawling()
    crawling_codimap.main_page_crawling()
```